server/storage/database_test.go

```diff
@@ -0,0 +1,278 @@
+package storage
+
+import (
+	"database/sql"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/go-sql-driver/mysql"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMySQLUpdateCurrent(t *testing.T) {
+	db, err := sqlmock.New()
```

Comment:
It's not the best mocking library, but allows me to simulate errors. Would like to work out a good way to run the happy paths against a real database so I can independently query the DB to ensure the data was inserted correctly